zigpy_deconz/api.py

```diff
@@ -63,7 +63,12 @@
     'aps_extended_panid': (0x0B, t.uint64_t),
     'trust_center_address': (0x0E, t.uint64_t),
     'security_mode': (0x10, t.uint8_t),
-    'network_key': (0x18, t.uint8_t),
+    'network_key': (
```

Comment:
Maybe copy/import `fixed_list(16, t.uint8_t)` from zigpy? https://github.com/zigpy/zigpy/blob/70886c5ac1d06cf65dab89556af142738d9105cb/zigpy/types/basic.py#L213

Reply:
Not having a firmware file allowing to read the key, I rather chose the most simple way. In other words: I feel uncomfortable adding more code for something that's likely never going to get used.